lib/util.js

```diff
@@ -49,7 +50,9 @@ const util = {
       }
     })
 
-    const out = 'solutions = ' + JSON.stringify(solutions, replacer, 2)
+    if (config.gClientCacheDir) cache_dir_line = 'cache_dir = "' + config.gClientCacheDir + '"\n'
```

Comment:
I think this might be a little easier to follow if you start with `let out = ''` and then append things one at a time

Reply:
Fixed, see latest version